exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/UserSession.java

```diff
@@ -115,6 +118,7 @@ public UserSession build() {
 
   private UserSession() {
     queryCount = new AtomicInteger(0);
+    sessionId = nextSessionId.getAndIncrement();
   }
```

Comment:
As far as I know, UserSession instance is created for each session, won't this mean that after your changes each session will have the same id? Actually CTTAS are going to add session id in its scope (it's represented as String and generated using UUID). You make take a look at those changes - https://github.com/apache/drill/pull/666